src/test/java/controllers/dialogs/EditTopicInvalidTest.java

```diff
@@ -0,0 +1,85 @@
+package controllers.dialogs;
+
+import com.jfoenix.controls.JFXButton;
+import com.jfoenix.controls.JFXTextField;
+import javafx.scene.Scene;
+import javafx.scene.control.ButtonType;
+import javafx.scene.layout.AnchorPane;
+import javafx.stage.Stage;
+import model.datatypes.Topic;
+import org.junit.Before;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.Parameterized;
+import utils.JavaFXThreadingRule;
+
+import java.lang.reflect.Field;
+import java.util.Arrays;
+import java.util.Collection;
+
+import static junit.framework.TestCase.assertNull;
+
+/**
+ * Created by Gorka Olalde on 9/6/16.
+ */
+@RunWith(Parameterized.class)
+public class EditTopicInvalidTest {
+
+    private JFXTextField newValueField;
+
+    private JFXButton saveBtn;
+
+    private EditTopicDialog dialog;
+
+    private Stage stage;
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Perhaps 'stage' could be replaced by a local variable.](https://www.codacy.com/app/POPBL6/front-end/file/2720643974/issues/source?bid=3316499&fileBranchId=3347636#l35)